src/main/java/org/pac4j/jax/rs/pac4j/JaxRsApplicationLogoutLogic.java

```diff
@@ -0,0 +1,16 @@
+package org.pac4j.jax.rs.pac4j;
+
+import org.pac4j.core.engine.DefaultApplicationLogoutLogic;
+
+/**
+ * @author Victor Noel - Linagora
+ * @since 1.1.1
+ * 
+ */
+public class JaxRsApplicationLogoutLogic<C extends JaxRsContext> extends DefaultApplicationLogoutLogic<Object, C> {
```

Comment:
It's not exactly what I proposed indeed. Instead of the filter level, I had the logic level, but it should do the job as we have one logic in a filter. I imagine this setter in the filter:

```java
public void setProfileManagerFactory(final Function<C, ProfileManager> factory) {
   xxxLogic.setProfileManagerFactory(factory);
}
```